.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "prettier.singleQuote": true,
+  "prettier.printWidth": 75
+}
```

Comment:
Kérlek ne commitold a saját beállításaidat.

Reply:
Ok, ez sem szándékos